transaction-overview.md

```diff
@@ -37,6 +37,10 @@ START TRANSACTION WITH CONSISTENT SNAPSHOT;
 
 All of the above three statements are used to start a transaction with the same effect. You can explicitly start a new transaction by executing one of these statements. If the current session is in the process of a transaction when one of these statements is executed, TiDB automatically commits the current transaction before starting a new transaction.
 
+> **Note:**
+>
+> Unlike MySQL, TiDB takes a snapshot of the current database after executing the above statements. MySQL's `BEGIN` and `START TRANSACTION` takes the snapshot after executing the first `SELECT` statement (not `SELECT FOR UPDATE`) that reads data from InnoDB when a transaction is started. `START TRANSACTION WITH CONSISTENT SNAPSHOT` takes a snapshot during the execution of the statement. As a result, `BEGIN`, `START TRANSACTION` and `START TRANSACTION WITH CONSISTENT SNAPSHOT` are equivalent to `START TRANSACTION WITH CONSISTENT SNAPSHOT` in MySQL.
```

Comment:
```suggestion
> Unlike MySQL, TiDB takes a snapshot of the current database after executing the statements above. MySQL's `BEGIN` and `START TRANSACTION` take a snapshot after executing the first `SELECT` statement (not `SELECT FOR UPDATE`) that reads data from InnoDB after a transaction is started. `START TRANSACTION WITH CONSISTENT SNAPSHOT` takes a snapshot during the execution of the statement. As a result, `BEGIN`, `START TRANSACTION`, and `START TRANSACTION WITH CONSISTENT SNAPSHOT` are equivalent to `START TRANSACTION WITH CONSISTENT SNAPSHOT` in MySQL.
```

Reply:
Two `after` are used consecutively. I can't think of a better way to put it, so let's keep this one for now.